nats/pkg/command/server.go

```diff
@@ -37,9 +37,10 @@ func Server(cfg *config.Config) *cli.Command {
 
 			natsServer, err := nats.NewNATSServer(
 				ctx,
+				logging.NewLogWrapper(logger),
 				nats.Host(cfg.Nats.Host),
 				nats.Port(cfg.Nats.Port),
-				nats.Logger(logging.NewLogWrapper(logger)),
+				nats.ClusterID("ocis-cluster"),
```

Comment:
Imho the `ClusterID` should come from configuration

Reply:
fixed